src/components/OTP.vue

```diff
@@ -212,8 +237,18 @@ export default {
       return this.programData.text.default.invalid.input;
     },
 
-    /** Displays message returned from OTP service */
+    /** Whether OTP message will be displayed.
+     * Will not be displayed if the timer is finished. Will be replaced by the resend OTP button
+     */
     willDisplayOTPMessage() {
```

Comment:
Also, can you change the name here?
Again applying the same principle of using `is` for booleans, `isOTPMessageDisplayed`